crates/networking/rpc/engine/payload.rs

```diff
@@ -34,7 +35,14 @@ impl RpcHandler for NewPayloadV1Request {
 
     async fn handle(&self, context: RpcApiContext) -> Result<Value, RpcErr> {
         validate_execution_payload_v1(&self.payload)?;
-        let block = get_block_from_payload(&self.payload, None, None)?;
+        let block = match get_block_from_payload(&self.payload, None, None) {
```

Comment:
nice! Can we see if we can enable some engine tests too? https://github.com/lambdaclass/ethrex/issues/2041 there are some related to invalid payload & transactions

Reply:
The InvalidPayload tests with the Syncing flag set to False pass, but the ones with Syncing=True do not and we cannot filter them out